src/Model/Position.php

```diff
@@ -155,7 +155,7 @@ public function setCreatedAt($createdAt)
      *
      * @return \DateTime
      */
-    public function getUpdatedAt()
+    public function getUpdatedAt(): \DateTime
```

Comment:
If the row has never been updated does it return the created at or a null?

Reply:
good point. I'll ?nullable it